src/main/java/de/sakpaas/backend/dto/UserInfoDto.java

```diff
@@ -0,0 +1,19 @@
+package de.sakpaas.backend.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+@Getter
+@Setter
```

Comment:
Setter für Dtos eigentlich nur in Ausnahmefällen.
Die werden ja idr. nur einmal über den Constructor erstellt und sollten dann nicht mehr geändert werden